package.json

```diff
@@ -1,63 +1,66 @@
 {
-    "name": "Brackets",
```

Comment:
Sorry, but we need to keep the 4-space indents, or rebasing on Adobe's stuff is going to suck.  Can you revert that bit and just add your actual changes?